src/_guides/language/language-tour.md

```diff
@@ -3136,20 +3136,20 @@ abstract class Musical {
 {% endprettify %}
 
 <div class="alert alert-info" markdown="1">
-**Note:**
-As of 1.13, two restrictions on mixins have been lifted
-from the Dart VM:
+  **Note:**
+  As of 1.13, two restrictions on mixins have been lifted
+  from the Dart VM:
 
-* Mixins allow extending from a class other than Object.
-* Mixins can call `super()`.
+  * Mixins allow extending from a class other than Object.
+  * Mixins can call `super()`.
 
-These "super mixins" are
-[not yet supported in dart2js](https://github.com/dart-lang/sdk/issues/23773)
-and require the `--supermixin` flag in dartanalyzer.
+  These "super mixins" are
```

Comment:
This paragraph needs work. `--supermixin` is no longer supported, right? And will dart2js ever support super mixins? (The referenced bug has been closed.) Is the upshot that only Flutter and the Dart VM can use super mixins, and only until the syntax is fixed? https://github.com/dart-lang/sdk/issues/23770 has more info.

Reply:
> --supermixin is no longer supported, right?

I got the impression from @eernstg's comment that the plan is to deprecate it in Dart 2.

Maybe we can just drop this side note?

Reply:
sgtm

Reply:
Actually, though... Flutter does make use of super mixins, doesn't it? If it's only the Flutter implementation, I'm OK with dropping this note. If it's Flutter users, we need to say something in case they have . Maybe we can just refer to the fact that the restrictions on mixins are likely to change. Something like this:

**Note:** Some restrictions on mixins are being removed. For details, see [SDK issue 23770](https://github.com/dart-lang/sdk/issues/23770).

Reply:
Updated to:

>   **Note:** Some restrictions on mixins are being removed. For details, see the [proposed mixin specification.][Dart 2.0 Mixins]

  [Dart 2.0 Mixins]: https://github.com/dart-lang/sdk/blob/master/docs/language/informal/mixin-declaration.md